cv_next/app/cv/[cvId]/components/commentSection/comment.tsx

```diff
@@ -1,78 +1,248 @@
 "use client";
+
 import { deleteComment } from "@/app/actions/comments/deleteComment";
 import { setResolved } from "@/app/actions/comments/setResolved";
 import { upvoteComment } from "@/app/actions/comments/setLike";
 import { addComment } from "@/app/actions/comments/addComment";
-import { mutate } from "swr";
+import { useSWRConfig } from "swr";
 import { RxPlus } from "react-icons/rx";
 import { GoCheckCircle } from "react-icons/go";
 import { GoCheckCircleFill } from "react-icons/go";
 import { FaRegTrashCan } from "react-icons/fa6";
 import { FaComment } from "react-icons/fa";
 import Tooltip from "../../../../../components/ui/tooltip";
-
 import { AiTwotoneLike } from "react-icons/ai";
 import { AiFillLike } from "react-icons/ai";
-
 import { useState } from "react";
 import Alert from "../../../../../components/ui/alert";
-import { useRouter } from "next/navigation";
-
 import Link from "next/link";
-import { useSupabase } from "@/hooks/supabase";
 
-export default function Comment({
-  comment,
+const NewCommentBlock = ({
+  commentOnCommentStatus,
+  setCommentOnComment,
+  addNewCommentClickEvent,
+}: any) => {
```

Comment:
I know it's a bit tiresome, but please don't leave `any` type.
You can utilize the `CommentProps` interface with `Omit<>`.